tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -434,6 +434,8 @@ build_rootfs_distro()
 			${engine_build_args} \
 			--build-arg http_proxy="${http_proxy}" \
 			--build-arg https_proxy="${https_proxy}" \
+			--build-arg RUST_TOOLCHAIN="$(get_package_version_from_kata_yaml  ".languages.rust.meta.newest-version")" \
+			--build-arg GO_VERSION="$(get_package_version_from_kata_yaml  ".languages.golang.version")" \
```

Comment:
I think @gkurz would ask if this commit can be merged into the first one in case of a bisect that might split them and I don't think they work independently?

Reply:
It can, right this would make more sens since those two are "atomic".